root/io/TFile/execNestedCollectionStreamerInfo.cxx

```diff
@@ -0,0 +1,50 @@
+#include <vector>
+#include "TTree.h"
+#include "TFile.h"
+#include "TMemFile.h"
+#include <iostream>
+#include <memory>
+
+struct Inside {
```

Comment:
Could you add a comment why we don't need a dictionary for any of these classes?